robottelo/hosts.py

```diff
@@ -191,13 +192,15 @@ def os_version(self):
         return Version(version=version_string)
 
     def setup(self):
-        self.remove_katello_ca()
-        self.execute('subscription-manager clean')
+        if not self.blank:
+            self.remove_katello_ca()
+            self.execute('subscription-manager clean')
```

Comment:
Is this needed because of some state where a katello_ca isn't present causing issues?

Reply:
see https://github.com/SatelliteQE/robottelo/pull/9246/files#r929309510